can-observation.js

```diff
@@ -220,13 +234,17 @@ canReflect.assignSymbols(Observation.prototype, {
 	},
 	"can.setPriority": function(priority){
 		this.options.priority = priority;
-	},
-	//!steal-remove-start
-	"can.getName": function() {
-		return canReflect.getName(this.constructor) + "<" + canReflect.getName(this.func) + ">";
 	}
-	//!steal-remove-end
-});
+};
+
+//!steal-remove-start
+if (process.env.NODE_ENV !== 'production') {
+	observationProto[canSymbol.for("can.getName")] = function() {
```

Comment:
Using canSymbol.for() is not needed here because `observationProto` is passed into assignSymbols().